js/modules/k6/http/http.go

```diff
@@ -92,7 +92,10 @@ func (r *RootModule) NewModuleInstance(vu modules.VU) modules.Instance {
 		args = append([]goja.Value{goja.Undefined()}, args...) // sigh
 		return mi.defaultClient.Request(http.MethodGet, url, args...)
 	})
-	mustExport("head", mi.defaultClient.getMethodClosure(http.MethodHead))
+	mustExport("head", func(url goja.Value, args ...goja.Value) (*Response, error) {
+		args = append([]goja.Value{goja.Undefined()}, args...) // sigh
```

Comment:
```suggestion
        // It avoid creating a dedicated method reusing the common Request method
        // setting the body of the request to null.
		args = append([]goja.Value{goja.Undefined()}, args...)
```
Sorry for seeing it just now, but if we want a comment there then I think we should use the opportunity for explaining why we are doing it.